docker-sync.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+options:
+  compose-file-path: './docker-compose-dev.yml'
+syncs: 
+  pas_dev_test-ehr-sync:
+    src: ../test-ehr
+    sync_excludes: ['.gradle', 'build', 'target', 'logs']
```

Comment:
If I am missing any other files/directories that should not be synced between the local machine and container, let me know. Those files need to be specified here and in the docker-compose.yml volumes (not specifying in volume will result in the sync strategy removing the file/folder from the container all together) for all repos

Examples include: build files, binaries, dependencies, database files, etc.